src/controllers/userController.js

```diff
@@ -106,6 +106,146 @@ class UserController {
       });
     }
   }
+
+  /**
+ *
+ *@description Follow User
+ * @static
+ * @param {object} req
+ * @param {object} res
+ * @returns {object} res
+ * @memberof UserController
+ */
+  static async followUser(req, res) {
+    try {
+      const { userId } = req.params;
+      const errors = {};
+
+      const findUser = await User.findOne({
+        where: {
+          id: userId
+        }
+      });
+
+      if (findUser !== null) {
+        const findFollow = await Follow.findOne({
+          where: {
+            followId: userId
+          }
+        });
+
+        const token = req.user;
+        // CHECK IF USER IS TRYING TO FOLLOW THEMSELVES
+        if (token !== Number(userId)) {
+        // IF USER IS NOT ALREADY FOLLOWING
+          if (findFollow === null) {
+            await Follow.create({
+              userId: token,
+              followId: userId
+            });
+
+            // RETURN SUCCESS IF SUCCESS
+            return res.status(200).json({
+              status: 200,
+              data: 'you just followed this user',
```

Comment:
Which user? Please use a better (more descriptive message here)

Reply:
Update the rest of your response messages in this file to be more descriptive and well-formatted.